code/mission/missionparse.h

```diff
@@ -94,13 +94,20 @@ typedef struct support_ship_info {
 } support_ship_info;
 
 // movie type defines
-#define	MOVIE_PRE_FICTION		0
-#define	MOVIE_PRE_CMD_BRIEF		1
-#define	MOVIE_PRE_BRIEF			2
-#define	MOVIE_PRE_GAME			3
-#define	MOVIE_PRE_DEBRIEF		4
-#define	MOVIE_POST_DEBRIEF		5
-#define	MOVIE_END_CAMPAIGN		6
+// If you add one here, you must also add a description to missioncutscenedlg.cpp for FRED
+// and update the dropdown list in fred.rc for the mission cutscene dialog editor.
+enum : int {
+	MOVIE_PRE_FICTION,
+	MOVIE_PRE_CMD_BRIEF,
+	MOVIE_PRE_BRIEF,
+	MOVIE_PRE_GAME,
+	MOVIE_PRE_DEBRIEF,
+	MOVIE_POST_DEBRIEF,
+	MOVIE_END_CAMPAIGN,
+
+	// Must always be at the end of the list
+	First_available_movie_id
```

Comment:
`First_available_movie_id` is misleading because unlike sexp operators, there isn't a way for modders to add additional movie types.  Change this to something like `Num_movie_ids` or `Movie_id_count`.

Reply:
Really? Ok. I'm just closing the PR. Not worth it.